code/script.js

```diff
@@ -0,0 +1,107 @@
+const projects = document.getElementById('projects')
+const profile = document.getElementById('profile')
+const repos = document.getElementById('repos')
+
+const username = 'StephannieMedenilla'
+const USER_PROFILE = `https://api.github.com/users/SteppbySteph`
+const USER_REPO = `https://api.github.com/users/SteppbySteph/repos`
+
+
+
+const options = {
+    method: 'GET',
+    headers: {
+          Authorization: `token ${GITHUB_TOKEN}`
+    }
+}    
```

Comment:
Nice work to implement the token and not give up even if obstacles during the "road". For the Authorization part, you don't have to have template literals - I saw that Karin had it in here code also - not really sure why and the reason for having the word "token" before the API const name, do you? Otherwise you've could just have written: 
```suggestion
const options = {
    method: 'GET',
    headers: {
          Authorization: 'GITHUB_TOKEN'
    }
}    
```